src/app/pages/storage/volumes/volumes-list/volumes-list.component.ts

```diff
@@ -1064,6 +1064,7 @@ export class VolumesListTableConfig implements InputTableConf {
           name: T('Add Zvol'),
           label: T("Add Zvol"),
           onClick: (row1) => {
+            console.log('addZvol', rowData.id);
```

Comment:
Leftover debug?

Reply:
Yes. After the review by Damian, I was making further changes. Will clean it up.